Userland/Libraries/LibJS/Tests/builtins/Intl/NumberFormat/NumberFormat.prototype.format.js

```diff
@@ -1019,3 +1009,57 @@ describe("style=unit", () => {
         expect(ja.format(123)).toBe("123km/h");
     });
 });
+
+describe("bigint", () => {
+    test("default", () => {
+        const en = new Intl.NumberFormat("en");
+        expect(en.format(1n)).toBe("1");
+        expect(en.format(12n)).toBe("12");
+        expect(en.format(123n)).toBe("123");
+
+        const ar = new Intl.NumberFormat("ar");
+        expect(ar.format(1n)).toBe("\u0661");
+        expect(ar.format(12n)).toBe("\u0661\u0662");
+        expect(ar.format(123n)).toBe("\u0661\u0662\u0663");
```

Comment:
can we add a test for a truly *big* integer here?

Reply:
Done!